ui/shared/components/panel/variants/SpliceOutlierLabel.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Label, Popup } from 'semantic-ui-react'
+
+import { getSortedIndividualsByFamily, getRnaSeqSignificantJunctionData } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING } from 'shared/utils/constants'
+import RnaSeqJunctionOutliersTable, { JUNCTION_COLUMN, OTHER_SPLICE_COLUMNS } from 'shared/components/table/RnaSeqJunctionOutliersTable'
+
+const INDIVIDUAL_NAME_COLUMN = { name: 'individualName', content: '', format: ({ individualName }) => (<b>{individualName}</b>) }
+
+const RNA_SEQ_SPLICE_POPUP_COLUMNS = [
+  INDIVIDUAL_NAME_COLUMN,
+  { ...JUNCTION_COLUMN, format: null },
+  ...OTHER_SPLICE_COLUMNS,
+]
+
+const HOVER_DATA_TABLE_PROPS = { basic: 'very', compact: 'very', singleLine: true }
+
+const BaseSpliceOutlierLabels = React.memo((
+  { variant, significantJunctionOutliers, individualsByFamilyGuid },
+) => {
+  const { pos, end, familyGuids, endChrom } = variant
+  const individualGuids = familyGuids.reduce((acc, fGuid) => (
+    [...acc, ...individualsByFamilyGuid[fGuid].map(individual => individual.individualGuid)]
+  ), [])
+
+  const overlappedOutliers = individualGuids.reduce((acc, iGuid) => (
+    [...acc, ...(significantJunctionOutliers[iGuid] || [])]
+  ), []).filter((outlier) => {
+    const { start: junctionStart, end: junctionEnd } = outlier
+    if ((pos >= junctionStart - RNASEQ_JUNCTION_PADDING) && (pos <= junctionEnd + RNASEQ_JUNCTION_PADDING)) {
+      return true
+    }
+    if (end && !endChrom) {
+      return (end >= junctionStart - RNASEQ_JUNCTION_PADDING) && (end <= junctionEnd + RNASEQ_JUNCTION_PADDING)
+    }
+    return false
+  })
```

Comment:
Should we use a selector to get the overlapped outliers of the variant?

Reply:
I think you should abstract the logic for getting overlapped junctions into a helper function, but that should be done in the IGV PR. The pre-grouping in the selector by family and chromosome (similar to what the  `VariantLocusListLabels` uses) will make that pretty efficient